docs/databases/backup.rst

```diff
@@ -0,0 +1,109 @@
+Docker database backup
+======================
+
+This page explains how to automate the backup process of a Docker database on a daily basis and restore the backup snapshot created by the `backup script <../../scripts/backup.sh>`_.
+
+   Note: This documentation assumes that you are using a Linux-based system.
+-------------------------------------------------------------------------------
+
+Here's how the script works
+---------------------------
+
+The script automates the process of creating PostgreSQL database backups from a Docker container. It generates a backup file(``.dump``) using the pg_dump utility in PostgreSQL and stores these files in ``/home/$USER/care-backups.`` which is binded to ``/backups`` in the docker container. Backup files older than 7 days are deleted when the script is executed. The backup file is saved with the name ``care_backup_%Y%m%d%H%M%S.sql``.
+
+Set up a cronjob
+----------------
+
+Backup your database running on docker automatically everyday by initiating a cronjob.
+
+    **Note**: Make sure you are in the care directory at the time of setting this up
```

Comment:
I don't see anything which requires a particular working directory here. Did you encounter problems?

It's not a bad place to stand, I'm just not sure it's a requirement.

Reply:
when tried executing the script from the user directory, it gave a permission error, but you are right we dont need to be inside the care dir